### ISSUE TITLE
Add support for multi repos

### ISSUE THREAD

**Issue by drhip** [Levi-Lesches/blocking-issues#3]
It doesn't work if we add link to another repo

**Comment by Levi-Lesches** [comment]
That's because the action updates itself by listening to the repo. From the example workflow file: 

```YAML
on: 
  issues:
    types: [closed]
  pull_request_target: 
    types: [opened, edited]
```

The action listens for new or edited PRs, parses out any blocking issues then waits. When an issue is closed, it checks if it was blocking any PRs and updates its status. If it can't listen for newly closed issues, this won't work, and PRs will forever stay blocked. If you could mark a PR as being blocked by an issue in another repo, the Action wouldn't have permissions to check that repo and you'd end up in the same trap. Alternatively, even if you did have permissions to both repos, it would still be infeasible to check every repo it knows about whenever any issue is closed. 

Cool use case, but sadly not supported. Closing, but feel free to continue or reopen with more feedback.

**Comment by LukeParky** [comment]
Is there some way to link it to every repo in a github project? I imagine it might not be possible but wanted your opinion

**Comment by Levi-Lesches** [comment]
I actually just had that use case myself! I have a project with multiple repos representing different hardware applications, and some issues in the higher level repos need to wait for the lower level code to be in place first. Then, in the project view, we can filter on `-label:blocked` to see only the tasks we can work on. This bot would be very helpful as we could mark issues as blocked and then they'll reappear in the project view when available. 

I'll add it to my to-do list, although I have some other priorities right now (including said project). It should be easier to do this than general cross-repo support, depending on what information the the projects API exposes.

**Comment by LukeParky** [comment]
That would be great, would you create a new issue for this or re-open this one? I'd like to subscribe to it so I can use this when I can. I also don't have room for it with other priorties right now.

**Comment by Levi-Lesches** [comment]
Opened #20 to track this.
